src/content/documentation/publish/source-code-submission.md

```diff
@@ -84,6 +84,12 @@ Code submission process happens after the add-on validation step.
 
 ![Upload sources screenshot](/assets/img/publish/upload-process-sources.png)
 
+#### Uploading source to an existing version
+
+You can also upload the source on an existing addon version, under "Manage Status & Versions".
+
```

Comment:
```suggestion
#### Upload source for an existing version

You can add source code to an existing add-on version. To do this, open [My Add-ons](https://addons.mozilla.org/en-US/developers/addons) and your add-on. In the left navigation, click **Manage Status & Versions**, then open the version you want to add source code to. At the bottom of the editor, next to **Source code**, click **Browse** and select the source code file. Make any other changes and then click **Save changes**.

```

I recommend not including the screenshot. (It takes up a lot of space, and without a description, it isn't easy to figure out what you're supposed to be looking at, and with the description, it's probably unnecessary.)